vendor/tracy.cmake

```diff
@@ -0,0 +1,25 @@
+if (MLN_USE_TRACY)
+    add_definitions(-DTRACY_ENABLE)
+endif()
+
+include(FetchContent)
+
+FetchContent_Declare(
+    tracy
+    GIT_REPOSITORY "https://github.com/wolfpld/tracy.git"
+    GIT_TAG master
```

Comment:
Should we use a release tag instead?

Reply:
Tracy author recommends master. There's no release tag but there are tags for specific release versions.

Reply:
Alright.